chaincode-labs/chaincode-podcast/2020-11-30-Carl-Dong-and-Build-Systems-Episode-9.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Carl Dong and Build Systems - Episode 9
+transcript_by: Whisper AI & PyAnnote
+categories: ['podcast']
+tag: ['GUIX talk at 2019 Breaking Bitcoin about reproducible builds', 'Gitian builds', 'Fake time', 'Timestamps and reproducibility of packages', 'reproducible-builds.org SOURCE_DATE_EPOCH', 'Toolchains', 'Windows Builds and reproducibility GCC libtool pointer confusion', 'NSIS', 'Using Debian as an example', 'User choice in their security model', 'Making the process accessible', 'Mac OS X Toolchain & SDK and cross-compiling']
+---
+
+Chaincode Labs podcast: Carl Dong and Build Systems - Episode 9
```

Comment:
for comparison: https://btctranscripts.com/chaincode-labs/chaincode-podcast/2020-11-30-carl-dong-reproducible-builds/